Dnn.AdminExperience/Dnn.PersonaBar.Extensions/Dnn.PersonaBar.Extensions.csproj

```diff
@@ -763,4 +763,764 @@
     <Error Condition="!Exists('..\..\packages\Yarn.MSBuild.1.16.0\build\Yarn.MSBuild.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\..\packages\Yarn.MSBuild.1.16.0\build\Yarn.MSBuild.targets'))" />
   </Target>
   <Import Project="..\..\packages\Yarn.MSBuild.1.16.0\build\Yarn.MSBuild.targets" Condition="Exists('..\..\packages\Yarn.MSBuild.1.16.0\build\Yarn.MSBuild.targets')" />
-</Project>
\ No newline at end of file
+</Project>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is causing the build to fail..

Reply:
Weird.  Every time I rebase or merge, it's duplication `csproj` files instead of actually merging.  I just pushed an updated commit.